### PR TITLE
fix reward ad event no call bug for ios

### PR DESCRIPTION
**Pull request by zhengbomo**

### CODE REVIEW

**Comment by zhengbomo** [comment]
https://github.com/bdlukaa/native_admob_flutter/issues/40

**Review by bdlukaa — CHANGES_REQUESTED** [review]
Does it work on Android?

File: lib/src/rewarded/rewarded.dart [hunk 1]
```diff
@@ -106,7 +106,7 @@ enum RewardedAdEvent {
 
 class RewardItem {
   /// Returns the reward amount.
-  final int? amount;
+  final double? amount;
```
Comment (bdlukaa) [inline lib/src/rewarded/rewarded.dart:109]: Wouldn't num be better?

Comment (zhengbomo) [reply lib/src/rewarded/rewarded.dart:109]: for ios amount is double value, for android is int
double is compatible with int，i think double will be better

**Comment by zhengbomo** [comment]
> Does it work on Android?

yes

**Pull request merged**